src/resolve/resolve-tool.c

```diff
@@ -41,7 +41,7 @@
 #include "strv.h"
 #include "terminal-util.h"
 
-#define DNS_CALL_TIMEOUT_USEC (90*USEC_PER_SEC)
+#define DNS_CALL_TIMEOUT_USEC SD_RESOLVED_QUERY_TIMEOUT_USEC
```

Comment:
please drop DNS_CALL_TIMEOUT_USEC and use the new macro wherever it was used before